component/container-runtime.jsonnet

```diff
@@ -0,0 +1,22 @@
+local common = import 'common.libsonnet';
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+
+local params = inv.parameters.openshift4_nodes;
+
+local containerRuntimeConfigs = [
+  kube._Object('machineconfiguration.openshift.io/v1', 'ContainerRuntimeConfig', nodeGroup) {
+    metadata+: {
+      labels+: common.DefaultLabels,
+    },
+    spec: params.containerRuntimeConfigs[nodeGroup],
+  }
+  for nodeGroup in std.objectFields(params.containerRuntimeConfigs)
+  if params.containerRuntimeConfigs[nodeGroup] != null
+];
```

Comment:
Looks good. Just as a future reference in case you didn't know, there is also the [`generateResources`](https://syn.tools/commodore/reference/commodore-libjsonnet.html#_generateresourcesresources_resourcefn) helper function in Commodore, which could have been used here. But the current solution is also good, no need to change it :)

Reply:
Didn't know about that, will definitely use this in the future 👍